src/scene_teleport.cpp

```diff
@@ -42,7 +43,7 @@ void Scene_Teleport::Update() {
 	teleport_window->Update();
 
 	if (Input::IsTriggered(Input::DECISION)) {
-		Game_System::SePlay(Game_System::GetSystemSE(Game_System::SFX_UseItem));
+		Game_System::SePlay(skill->sound_effect);
```

Comment:
This causes a crash. It needs to be moved down into the `if (skill)` block.

Reply:
And in the item block, we need to do `ReaderUtil::GetElement(Data::skills, item->skill_id)` and do the skill SFX there.